scoville/command.py

```diff
@@ -139,5 +139,63 @@ def proxy(url, port):
     serve_http(url, port)
 
 
+def read_urls(file_name, url_pattern):
+    with open(file_name, 'r') as fh:
+        for line in fh:
+            zxy = line.split(' ', 1)[0]
+            z, x, y = map(int, zxy.split('/', 2))
+
+            u = url_pattern \
+                .replace('{z}', str(z)) \
+                .replace('{x}', str(x)) \
+                .replace('{y}', str(y))
+
+            yield u
+
+
+@cli.command()
+@click.argument('tiles_file', required=1)
+@click.argument('url', required=1)
+@click.option('--percentiles', '-p', multiple=True, type=float,
+              help='Percentiles to display. Use decimal floats, i.e: 99.9, '
+              'not 99_9. Can be used multiple times.')
+@click.option('--cache/--no-cache', default=False, help='Use a cache for '
```

Comment:
Why specify `--no-cache` here if the absence of the option implies that? Or does it force clear the cache?

Reply:
That's just how the [click library](https://click.palletsprojects.com/en/7.x/options/#boolean-flags) does boolean flags. The default is `--no-cache`, but it can be specified explicitly if people want to be sure.